src/styles/partials/components/_alert.scss

```diff
@@ -3,17 +3,6 @@
   padding: $padding-large $default-padding;
```

Comment:
This rule needs to change a bit for icons https://github.com/baltimorecounty/dotgov-components/blob/5153da4d060050c991005d24fda31aa87483f756/src/styles/partials/components/_alert.scss#L37

font-size should be removed (it will inherit it from it's container) and the right padding should change from `20px` to `10px`

Reply:
another change in this file is on small screens and below the padding for `.dg_alert` should change to `10px` for the top and bottom instead of `20px` which is correct for screens above the small size.

Reply:
fyi, @sgrg1 looks like your branch is out of date

Reply:
@martypowell @tmccoy529 ,fixed issue with padding and merge with integration